src-python/drbench/drbench/drbench.py

```diff
@@ -62,7 +56,7 @@ def mhp(ctx, bench, fork, nprocs, sycl_cpu, sycl_gpu):
     if sycl_gpu:
         # mhp-bench will spread GPUs over ranks automatically, so no
         # pinning is needed
-        env = 'ONEAPI_DEVICE_SELECTOR=level_zero:gpu'
+        env = 'ONEAPI_DEVICE_SELECTOR=ext_oneapi_cuda:gpu'
```

Comment:
I suspect you did this so you could test on your own system. I would like to benchmark on cuda as well so something more permanent would be useful.

Reply:
Yes. I've just remade the selector to have logical _or_ (select level_zero:gpu or ext_oneapi_cuda:gpu). Please check if I can commit to _main_ this new version of selector.